src/app/components/practice-container/practice-container.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { AnalyseDialogComponent } from '@components/analyse-dialog/analyse-dialog.component';
+
+@Component({
+  selector: 'app-practice-container',
+  templateUrl: './practice-container.component.html',
+  styleUrls: ['./practice-container.component.sass']
+})
+export class PracticeContainerComponent implements OnInit {
+
+  document = {
+    text: "Hello world",
+    phrases: [
+        {
+            startIndex: 0,
+            endIndex: 10,
+        },
+    ],
+  };
+
+
+  constructor(public dialog: MatDialog) { }
+
+  ngOnInit(): void {
+  }
+
+  public openDialog(): void {
+    const dialogRef = this.dialog.open(AnalyseDialogComponent);
+    dialogRef.afterClosed().subscribe(result => {
+      console.log(`Dialog result: ${result}`);
```

Comment:
Do we need this? What is the result of the afterClosed observable? (I get `undefined`)

Reply:
we don't need it for now. We might need this when we figure out waht we want to do when the dialog is closed. Will comment it out for now